.taskcluster.yml

```diff
@@ -2,123 +2,237 @@
 # License, v. 2.0. If a copy of the MPL was not distributed with this
 # file, You can obtain one at http://mozilla.org/MPL/2.0/.
 
-
 version: 1
 policy:
   # https://docs.taskcluster.net/docs/reference/integrations/taskcluster-github/docs/taskcluster-yml-v1#pull-requests
-  pullRequests: collaborators
-
+  pullRequests: public
 tasks:
-  - $if: 'tasks_for == "github-pull-request"'
-    then:
-      $if: 'event.action in ["opened", "reopened", "synchronize"]'
-      then:
-        taskGroupId: {$eval: as_slugid("decision_task")}
-        taskId: {$eval: as_slugid("decision_task")}
-        provisionerId: aws-provisioner-v1
-        workerType: application-services-r
-        created: {$fromNow: ''}
-        deadline: {$fromNow: '1 day'}
-        metadata:
-          name: "Application Services: GitHub decision task"
-          description: ""
-          owner: &task_owner ${event.pull_request.head.user.login}@users.noreply.github.com
-          source: &task_source ${event.pull_request.head.repo.url}
-        scopes:
-          - "queue:scheduler-id:taskcluster-github"
-          # So that we can use our own "application-services-r" worker-type.
-          - "queue:create-task:highest:aws-provisioner-v1/application-services-r"
-
-          # So that we can cache task outputs for re-use.
-          - "queue:route:index.project.application-services.*"
-
-          # So that we can re-use Gradle/Cargo/sccache bits between tasks.
-          - "docker-worker:cache:application-services-*"
+  $let:
+    decision_task_id: {$eval: as_slugid("decision_task")}
+    expires_in: {$fromNow: '1 year'}
```

Comment:
Uh, what does this concern?

Reply:
It's set by default to 1y, I assume it was set explicitely because of the chain of trust stuff